java/src/jmri/jmrit/vsdecoder/VSDecoderManager.java

```diff
@@ -332,10 +322,12 @@ public VSDecoder getVSDecoderByAddress(String sa) {
         return rv;
     }
 
+    @Deprecated
```

Comment:
It's good to add a comment about the deprecation with the JMRI version where the deprecation is done. That way, we know when it was done and when we can remove the method completely.

Se the bottom of the section `Deprecating Code` on this page:
https://www.jmri.org/help/en/html/doc/Technical/RP.shtml

Reply:
Will change that too.